README.rst

```diff
@@ -442,6 +442,74 @@ In order to switch to this test runner you have set the following in your `setti
 
     TEST_RUNNER = 'openwisp_utils.tests.TimeLoggingTestRunner'
 
+``openwisp_utils.tests.redirect_stdout``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This decorator can be used to redirect standard output produced by tests to somewhere else.
+
+Usage example as a decorator:
+
+.. code-block:: python
+
+    import io
+    from openwisp_utils.tests import redirect_stdout
+
+    @redirect_stdout(io.StringIO())
+    def some_test(self):
+        # Pseudo Code
+        print("some output")
+
+    @redirect_stdout(io.StringIO())
+    def some_test(self, stdout):
+        # Pseudo Code
+        self.assertIn('expected stdout', stdout.getvalue())
+
+``openwisp_utils.tests.redirect_stderr``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Similar to redirect_stdout, this decorator can be used to redirect standard error produced by tests to somewhere else.
+
+Usage example as a decorator:
+
+.. code-block:: python
+
+    import io
+    from openwisp_utils.tests import redirect_stderr
+
+    @redirect_stderr(io.StringIO())
+    def some_test(self):
+        # Pseudo Code
+        print("this test will print some error")
+
+    @redirect_stderr(io.StringIO())
+    def some_test(self, stderr):
+        # Pseudo Code
+        self.assertIn('expected error', stderr.getvalue())
+
+``openwisp_utils.tests.redirect_any_output``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This decorator can be used to redirect any output produced by tests to somewhere else.
```

Comment:
I recommend make it more explicit: `any output (standard output and standard error)`